tests/libtest.sh

```diff
@@ -10,7 +10,18 @@ vg_exists() {
     fi
   done
   return 1
-  
+}
+
+# Tests if the logical volume lv_name exists
+lv_exists() {
+  local lv lv_name="$1"
+
+  for lv in $(lvs --noheadings -o lv_name); do
+    if [ "$lv" == "$lv_name" ]; then
```

Comment:
I used existing logic and structure of `vg_exists()` while developing `lv_exists()`. I wanted to keep `lv_exists()` as close as possible to existing code style. Does this work for you ? or should we change both (vg_exists, lv_exists) to incorporate this logic ?